interview-study-notes-for-security-engineering.md

```diff
@@ -397,12 +397,13 @@ I recommend doing a lot of reading and whenever you come across a term you are u
 # Malware & Reversing
 
 - Interesting malware
-	- Conficker. 
```

Comment:
I appreciate that you have added news articles for the malware, however I avoided doing this for a few reasons.
I want to encourage people to do their own research. I find that a list of links can lead to more distractions (death by too many open tabs). I don't want people to think these links are the authoritative source on the subject or to endorse those specific sources. I also want to avoid linking to external websites and needing to do maintenance / dead-link clean-up down the line. 

Thanks for the sunburst addition!

```suggestion
	- Conficker.
	- Morris worm.
	- Zeus malware.
	- Stuxnet.
	- Wannacry.
	- CookieMiner.
	- Sunburst.
```